view/frontend/web/js/view/payment/method-renderer/adyen-oneclick-method.js

```diff
@@ -184,6 +207,7 @@ define(
                          * @returns {boolean}
                          */
                         placeOrder: function (data, event) {
+                            debugger;
```

Comment:
Is it WIP?

Reply:
it was but it is working so now removed it.